From: Sven Roederer <devel-sven@geroedel.de>
Date: Wed, 28 Apr 2021 00:28:42 +0200
Subject: Revert "[WIP] ath79: Fix LAN-port for tplink_eap2x5-1port"

This reverts commit 82be70f4db90810ff3baa9e88483bb280bbdb4bb as it's not fixing the issue.
see also https://forum.openwrt.org/t/eap225-v1-lan-port-link-speed-100mbit-not-working/88060/8

diff --git a/target/linux/ath79/dts/qca9563_tplink_eap2x5-1port.dtsi b/target/linux/ath79/dts/qca9563_tplink_eap2x5-1port.dtsi
index 4d99c234d9fbb3c6240e3eab5d0686452d9b9fe3..3978be75e663e57fc2c76893feb17004e3f92cde 100644
--- a/target/linux/ath79/dts/qca9563_tplink_eap2x5-1port.dtsi
+++ b/target/linux/ath79/dts/qca9563_tplink_eap2x5-1port.dtsi
@@ -109,6 +109,7 @@
 &eth0 {
 	status = "okay";
 
+	phy-handle = <&phy4>;
 	phy-mode = "sgmii";
 	pll-data = <0x03000000 0x00000101 0x00001313>;
 
@@ -119,11 +120,6 @@
 	gmac-config {
 		device = <&gmac>;
 	};
-
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
 };
 
 &wmac {
